src/adv.py

```diff
@@ -1,104 +1,203 @@
 import random
 from room import Room
+from item import Item, Treasure, LightSource
 from player import Player
 from adv_save import load_results, save_results
 import os
 
 
+def parse_command(com_mand):
+    if com_mand == '':
+        return
+    cmd_words = com_mand.split()  # split into word list
 
-# global vars
-save_file = "adv_save.txt"
+    if str(cmd_words[0]).lower() == 'inventory':  # __ player inventory _
+        curr_player.list_inventory()
+        return
+
+
+    if (curr_player.curr_room.is_light) or (curr_player.has_light()) or (curr_player.curr_room.has_light()):
+
+        if (str(cmd_words[0]).lower() == 'get') or (str(cmd_words[0]).lower() == 'take'):        # __ get/take item __
+            if len(cmd_words) > 1:
+                item_name = str(cmd_words[1]).lower()
+                if curr_player.curr_room.in_room(item_name):
+                    curr_player.add_item(curr_player.curr_room.get_item(item_name))
+                    curr_player.curr_room.remove_item(item_name)
+                    return
+                else:
+                    print('There is no', item_name, 'here')
+                    return
+            else:
+                print('Get what?.......')
+                return
+
+
+
+    else:
+        print('Good luck finding that in the dark!')
+        return
```

Comment:
I would think about turning this nested if/else block into a function. As a general rule for myself in programming, I take note of nested if conditions and refactor them into function(s). Maybe think about making a helper file to hold functions like this.